webrtc.html

```diff
@@ -5827,9 +5827,11 @@ <h2>Methods</h2>
                       <p>Determine if negotiation is needed to transmit
                       <var>withTrack</var> in place of the sender's existing
                       track. Negotiation is not needed if <var>withTrack</var>
-                      is null. Ignore which <code>MediaStream</code> the track
-                      resides in and the <code>id</code> attribute of the track
-                      in this determination. If negotiation is needed, then
+                      is null or if no changes are required to the currently
+                      negotiated SDP offer/answer envelope. Ignore which
```

Comment:
Somewhere this paragraph needs to define "the currently negotiated SDP offer/answer envelope" before we have negotiated anything - should probably be = the local capabilities (what's in createOffer).